src/renderer/wg_engine/tvgWgShaderTypes.cpp

```diff
@@ -146,6 +146,8 @@ void WgShaderTypeGradient::updateTexData(const Fill::ColorStop* stops, uint32_t
     for (uint32_t i = 1; i < stopCnt; i++)
         if (sstops.last().offset < stops[i].offset)
             sstops.push(stops[i]);
+        else if (sstops.last().offset == stops[i].offset)
```

Comment:
I think the addition can be done outside the for loop?

Reply:
Nope.. thats becouse not onlu first or last offsets can be the same, but in a middle of the list. last() == current() of the filtered list